Android/app/src/main/res/layout/activity_welcome.xml

```diff
@@ -46,7 +46,7 @@
         android:layout_width="0dp"
         android:layout_height="75dp"
         app:tabMode="fixed"
-        android:background="@color/color_accent"
+        android:background="@color/color_primary"
```

Comment:
We should think about changing primary from purple to red. We'll discuss this on the next daily meeting.